microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -825,7 +827,33 @@ def _install_fedora_dependencies(self) -> None:
         if not self.use_package_manager_install():
             self._install_ninja_and_meson()
 
+    def _resize_home_folder(self) -> None:
```

Comment:
Can it be installed in other frees pace partition? Extend the home folder may not be appliable on all distros.

Reply:
Panfeng fixed a similar disk space issue on gpu test cases. Can we have a common method to handle this?